charts/vc-authn-oidc/templates/deployment.yaml

```diff
@@ -134,6 +143,10 @@ spec:
               mountPath: /opt/token
             - name: controller-config
               mountPath: /etc/controller-config
+            {{- if .Values.controller.htmlFileOverrides }}
```

Comment:
Toggle on `.Values.controller.customHtmlConfigMapName` like above, but definition does not need to change.